sonar-plugin-api/src/main/java/org/sonar/api/batch/sensor/issue/NewIssue.java

```diff
@@ -40,10 +40,17 @@
 
   /**
    * Effort to fix the issue.
+   * @deprecated since 5.5 use {@link #gap(Double)}
    */
+  @Deprecated
   NewIssue effortToFix(@Nullable Double effortToFix);
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)